app/src/main/res/drawable/ic_quest_camp_type.xml

```diff
@@ -1,16 +1,9 @@
-<vector
-    android:height="128dp"
-    android:viewportHeight="33.86667"
-    android:viewportWidth="33.86667"
-    android:width="128dp"
-    xmlns:android="http://schemas.android.com/apk/res/android">
+<vector xmlns:android="http://schemas.android.com/apk/res/android" xmlns:aapt="http://schemas.android.com/aapt"
```

Comment:
note that icon appears to be partially transparent - is it intentional and working well?

Reply:
Not particularly intentional, no. It's just the way icon came. Seems to be working just fine in menu and on map, though. Perhaps some other color than white would look nicer, if someone with inkscape skills could help (my attempt seem to create "borders" of some kind between the tent and background).

![small_Screenshot_20220803_141451_de westnordost streetcomplete mn debug](https://user-images.githubusercontent.com/156656/182605287-ed6383a2-4850-4cc1-97ab-c78545ece3c5.jpg) ![small_Screenshot_20220803_141445_de westnordost streetcomplete mn debug](https://user-images.githubusercontent.com/156656/182605293-86708caf-7fae-4bd4-912d-cdff95a2da81.jpg)

Reply:
> Seems to be working just fine in menu and on map, though

Yes, as it shown on top of nontransparent white pin, so it works out fine.